Terminal.Gui/Views/Dialog.cs

```diff
@@ -55,13 +55,11 @@ public class Dialog : Window
     private readonly List<Button> _buttons = new ();
 
     /// <summary>
-    ///     Initializes a new instance of the <see cref="Dialog"/> class using <see cref="LayoutStyle.Computed"/>
-    ///     positioning with no <see cref="Button"/>s.
+    ///     Initializes a new instance of the <see cref="Dialog"/> class.
     /// </summary>
     /// <remarks>
-    ///     By default, <see cref="View.X"/> and <see cref="View.Y"/> are set to <c>Pos.Center ()</c> and
-    ///     <see cref="View.Width"/> and <see cref="View.Height"/> are set to <c>Width = Dim.Percent (85)</c>, centering the
-    ///     Dialog vertically and horizontally.
+    ///     By default, <see cref="View.X"/>, <see cref="View.Y"/>, <see cref="View.Width"/>, and <see cref="View.Height"/> are set
+    ///     such that the dialog will be centered and no larger than 90% of the container's width and height.
```

Comment:
How about this wording to make it parse easier:
```csharp
/// <remarks>
/// By default, <see cref="View.X"/>, <see cref="View.Y"/>, <see cref="View.Width"/>, and <see cref="View.Height"/> are set such that the <see cref="Dialog" /> will be centered in its containing <see cref="View" /> and no larger than 90% of the containing <see cref="View" />'s <see cref="View.Width" /> and <see cref="View.Height" />.
/// </remarks>
```
However the reflow will line it up of course. I just put it on one line here so it'll reformat it to fit if pasted.

But I also am wondering if maybe we should be dogfooding the API's terminology here, as well?

Specifically, instead of talking about a containing view or a container, should we be saying `<see cref="SuperView" />`?

Reply:
Actually, for Modal's there is NO containing View. Currently they use `Driver.Screen` to bound them. @BDisp  and I are debating whether Modals can have a SuperView or not elsewhere. 

I have an instinct that there are use-cases where an app would want this, but am struggling to come up with a real example. Therefore, we'll leave it as-is for now and I'll change this API doc to say "Screen".

Reply:
One use case is for windows which, themselves, want to have a modal popup, such as in an MDI-style app.

I was actually originally doing something like that in SnapsInAZfs, but went another direction since the means of achieving it in v1 would have required a lot of code around cases I wasn't interested in trying to predict. 😅

Reply:
That's probably something we could do in a feature release, though, rather than taking the time to do it in V2 RTM, isn't it?